docs/tokens.md

```diff
@@ -0,0 +1,327 @@
+# **Tokens && Fees**
+
+## **Table of content**
+
+- [**Tokens \&\& Fees**](#tokens--fees)
```

Comment:
I would remove this since its duplicated.

Reply:
I know, but that's how MD generates TOC. I could remove it, but it might result in inconsistency down the line.

Reply:
ohhh I did not know that.